stock-filters/ChirpVillage/Block.py

```diff
@@ -23,3 +23,6 @@ def __init__(self, x, z, height, type):
         self.height = height
         self.type = type
         self.biome_id = -1  # undefined by default, set in calculateBiomeMap of BlockUtils
+    
+    def set_type(self, new_type):
```

Comment:
why is this needed? nothing is private. we can modify the type member directly.

Reply:
meant to take this out